website/pages/downloads/index.jsx

```diff
@@ -1,103 +1,99 @@
-            ],
-            os: 'linux',
-          },
-          {
-            label: 'Amazon Linux',
-            commands: [
-              'sudo yum install -y yum-utils',
-              'sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/AmazonLinux/hashicorp.repo',
-              'sudo yum -y install boundary',
-            ],
-            os: 'linux',
-          },
-        ]}
-        latestVersion={VERSION}
-        getStartedLinks={[
-          {
-            label: 'Install Boundary',
-            href:
-              'https://learn.hashicorp.com/tutorials/boundary/getting-started-install',
-          },
-          {
-            label: 'Introduction to Boundary',
-            href:
-              'https://learn.hashicorp.com/tutorials/boundary/getting-started-intro',
-          },
-          {
-            label: 'Start a Development Environment',
-            href:
-              'https://learn.hashicorp.com/tutorials/boundary/getting-started-dev',
-          },
-        ]}
-        logo={
-          <img
-            className={styles.logo}
-            alt="Boundary"
-            src={require('@hashicorp/mktg-logos/product/boundary/primary/color.svg')}
-          />
-        }
-        product="boundary"
-        tutorialLink={{
-          label: 'View Tutorial on HashiCorp Learn',
+    <ProductDownloadsPage
+      releases={binaryReleases}
+      packageManagers={[
+        {
+          label: 'Homebrew',
+          commands: [
+            'brew tap hashicorp/tap',
+            'brew install hashicorp/tap/boundary',
+          ],
+          os: 'darwin',
+        },
+        {
+          label: 'Ubuntu/Debian',
+          commands: [
+            'curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -',
+            'sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"',
+            'sudo apt-get update && sudo apt-get install boundary',
+          ],
+          os: 'linux',
+        },
+        {
+          label: 'CentOS/RHEL',
+          commands: [
+            'sudo yum install -y yum-utils',
+            'sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo',
+            'sudo yum -y install boundary',
+          ],
+          os: 'linux',
+        },
+        {
+          label: 'Fedora',
+          commands: [
+            'sudo dnf install -y dnf-plugins-core',
+            'sudo dnf config-manager --add-repo https://rpm.releases.hashicorp.com/fedora/hashicorp.repo',
+            'sudo dnf -y install boundary',
+          ],
+          os: 'linux',
+        },
+        {
+          label: 'Amazon Linux',
+          commands: [
+            'sudo yum install -y yum-utils',
+            'sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/AmazonLinux/hashicorp.repo',
+            'sudo yum -y install boundary',
+          ],
+          os: 'linux',
+        },
+      ]}
```

Comment:
Unless these commands are different than the standard set on purpose, they can be omitted, they are built in to the core